### ISSUE TITLE
Error: Unable to connect to undefined

### ISSUE THREAD

**Issue by zrrrzzt** [cfpb/node-wcag#26]
I haven't had the time to look into this, but here's what I've got so far.

``` sh
$ wcag https://www.mrfylke.no/ --id=69f6ea6a9e30b37c3d341d3b716df218f8942c0e
```

Returns

``` sh
Error: Unable to connect to undefined
```

The same error message is returned from the module.

``` javascript
'use strict';

var wcag = require('wcag');
var options = {
  id: '69f6ea6a9e30b37c3d341d3b716df218f8942c0e',
  uri: 'https://www.mrfylke.no',
  guide: 'WCAG2-AA'
};

wcag(options, function (error, data) {
  if (error) {
    console.error(error);
  } else {
    console.log(data);
  }
});

//=> [Error: Unable to connect to undefined]

```

So far I've only seen this error with this url. If you check the url via the achecker websites everything is ok. The error message itself seems to be returned from `lib/checkURI.js` so that's probably a good place to start.

**Comment by zrrrzzt** [comment]
Hmmm, looks like there's a problem with the website's certificate.

``` javascript
var http = require('http-https');
var uri = 'https://www.mrfylke.no/'


http.get(uri, function(data) {
  console.log(data);
}).on('error', function(e) {
  console.error('error: ' + e);
});

//=> error: Error: unable to verify the first certificate
```

So what to do?

I think we definitely should return a better error message. 
I'm not sure if we should fail or not. If we mimic the AChecker website, the correct solution would be to return the check anyway. But, as a webmaster I would appreciate to be informed if my certificates are bad.

**Comment by contolini** [comment]
Hmmm, good catch! I vote to check the website anyway.

**Comment by zrrrzzt** [comment]
+1
Not sure how the solve it though. Maybe find another way to check the connection?

**Comment by contolini** [comment]
@zrrrzzt After some fooling around, I think we can simply pass [`rejectUnauthorized = false`](https://nodejs.org/api/https.html#https_https_request_options_callback) to `http`:

```
var http = require('http-https');
var url = require('url');
var uri = 'https://www.mrfylke.no/';

var opts = url.parse(uri);
opts.rejectUnauthorized = false;

http.get(opts).on('error', function(e) {
  console.error('error: ' + e);
});
```

I'll whip up a PR.

**Comment by zrrrzzt** [comment]
awsm :+1:
